config/manager/nephe-controller.yaml

```diff
@@ -1,5 +1,10 @@
 ---
 apiVersion: v1
+kind: Namespace
+metadata:
+  name: system
```

Comment:
I believe the system will be replaced by nephe-system defined in kustomize. I have this because it was how we were doing and if you look at other yamls, they all have this system namespace. It should be a legacy from antrea-plus-system with antrea-plus as a prefix. We should test and try reduce this if nephe-system will be auto-generated even if we don't have this system, or even the namespace explicitly defined.